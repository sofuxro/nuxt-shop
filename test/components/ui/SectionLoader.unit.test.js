import { factoryMount } from '~/test/utils/factories'

import Component from '~/components/ui/SectionLoader'

let wrapper

beforeEach(() => {
    wrapper = factoryMount(Component, {})
})

afterEach(() => {
    wrapper.destroy()
})

describe('SectionLoader', () => {
    test('is fully functional', () => {
        // Initial snapshot
        expect(wrapper.element).toMatchSnapshot()
    })
})
