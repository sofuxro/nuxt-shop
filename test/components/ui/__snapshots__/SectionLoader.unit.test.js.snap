// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionLoader is fully functional 1`] = `
<div
  class="section-loader-wrapper"
>
  <div
    class="loader"
  />
</div>
`;
